misc/version_updater.py

```diff
@@ -70,6 +74,7 @@ class Tool(enum.Enum):
     Tool.Node: "18.12.0",
     Tool.WasmPack: "0.11.0",
     Tool.Parsec: "2.16.0-a.0+dev",
+    Tool.License: "BUSL-1.1 OR AGPL-3.0-only",
```

Comment:
@touilleMan I've choose to use the value `BUSL-1.1 OR AGPL-3.0-only`, should I keep it that way or you prefer to directly move to `BUSL-1.1` only ?

Reply:
Yeah move on BUSL-1.1 directly

Reply:
So I need to also update the headers ?